### ISSUE TITLE
Addition of 'V=1' to %make_build broke a package

### ISSUE THREAD

**Issue by jasontibbitts** [rpm-software-management/rpm#798]
One of the packages I help maintain (apcupsd) broke oddly in rawhide with errors like this:
```
/bin/sh: 1man: command not found
make[3]: 1g++: Command not found
```
Turns out this comes from the `V=1` which was added to `%make_build` in 8655493bdfd6b76271893b148033f2ff580d2d39.  The software has an autoconf/targets.mak file which makes use of $(V) in a way that I guess differs from what's expected:  https://sourceforge.net/p/apcupsd/svn/HEAD/tree/branches/Branch-3_14/autoconf/targets.mak

I think that this usage is sufficiently weird that I wouldn't ask RPM to back this out.  But is there any chance that we could move the `V=1 VERBOSE=1` into `%_make_verbose` or something like that, just to simplify the workaround?

**Comment by pmatilai** [comment]
Fixed in commit c7e0b61c05878868300653d5892425c6c41fdba0

**Comment by pmatilai** [comment]
Meh, wrong button... Closing as per above.
